stagifyfrontend/src/app/services/authentication/authentication/authentication.service.ts

```diff
@@ -14,7 +14,7 @@ export class AuthenticationService {
   constructor(private httpClient:HttpClient, private cookieService:CookieService) { }
 
   isLoggedIn():boolean{
-      return this.cookieService.check('sessionid');
+      return this.cookieService.check('ARRAffinity');
```

Comment:
This is wrong I believe. `ARRAffinity` is a cookie by Azure (https://azure.github.io/AppService/2016/05/16/Disable-Session-affinity-cookie-(ARR-cookie)-for-Azure-web-apps.html).
The default name of the session cookie is indeed `sessionid` (https://docs.djangoproject.com/en/3.1/ref/settings/#session-cookie-name)